src/output.cpp

```diff
@@ -2341,6 +2341,114 @@ int msgtype_to_tilecolor( const game_message_type type, const bool bOldMsg )
     return -1;
 }
 
+bool wildcard_match(const std::string &sTextIn, const std::string &sPatternIn)
+{
+    std::string sText = sTextIn;
+    std::string sPattern = sPatternIn;
+
+    //case insenitive search
+
+    /* Possible patterns
```

Comment:
This kind of comment belongs into the function documentation in the header. None of those functions are currently documented at all.